content/parts/en/contributor-form.mdx

```diff
@@ -0,0 +1,15 @@
+## Sign your contribution
+
+Open Terms Archive is an open source initiative that relies on contributors just like you 🤗
+
+By default, your contributions will be anonymous. Signing your contributions enables you to be credited publicly for them and to be notified when they are integrated or discussed.
+
+<ContributorFormFields />
+
+- While not readable anywhere on the web, this email address will be [publicly](https://stackoverflow.com/a/897604/594053) and irreversibly associated with your contribution.
```

Comment:
> While not readable anywhere on the web

I don’t think that’s true? Or am I missing something?

The repository is hosted on GitHub, and GitHub displays the Git commit information like author and author email fields as well as the commit message.

Reply:
You're right on the fact that it can be read, but on the commit page through, I could not find any place mentioning the email.

![Screen Shot 2022-11-14 at 11 36 33](https://user-images.githubusercontent.com/4191809/201601572-830aaa3b-0bae-4b24-892d-e3cc98aad6ec.png)

It's available through an API call though.

<img width="725" alt="Screen Shot 2022-11-14 at 11 37 59" src="https://user-images.githubusercontent.com/4191809/201601846-2771aa88-577d-45fa-8a56-bbb15cf9e14b.png">

Reply:
Indeed, thanks @martinratinaud for the examples 🙂 

Hence my phrasing: this is not available on the web, but is public. Anyone cloning the repo can find it. Not through a web search though, and this will not be indexed.